scalactic-test/src/test/scala/org/scalactic/TypeCheckedTraversableEqualityConstraintsSpec.scala

```diff
@@ -33,7 +33,7 @@ class TypeCheckedTraversableEqualityConstraintsSpec extends FunSpec with NonImpl
   val sub1: Sub = new Sub(1)
   val super2: Super = new Super(2)
   val sub2: Sub = new Sub(2)
-  val nullSuper: Super = null
+  val nullSuper: Super | Null = null
 
   case class Fruit(name: String)
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.